### ISSUE TITLE
Confused

### ISSUE THREAD

**Issue by rchl** [davidolrik/sublime-rsync-ssh#30]
Sorry about bad summary but I'm just confused after trying this plugin. :)

- Documentation talks about 'Initialize Settings' menu option which doesn't seem to exist.
- "Example .sublime-project file" section has syntax errors in the example code
- Bundled RSync.sublime-settings file has a lot of options that don't seem to match those in exmaple project file. Like for example strsync.local_path which seems equivalent to path defined in "remotes" section of the example project file, but seems it is still required for plugin to operate (judging from the code which checks for 'local_path' pref).

So basically it seems to be quite broken on first sight, for someone who doesn't go beyond documentation. Are these actual bugs or I'm missing something?

**Comment by davidolrik** [comment]
I totally get why you are confused, the files toy are talking about is from another [plugin](https://github.com/gcarvalho/RSync) entirely.

**Comment by rchl** [comment]
Jeesh... Sorry. :)
I'll try yours instead. :)

**Comment by davidolrik** [comment]
No worries =)

**Comment by rchl** [comment]
But documentation here has some syntax errors (missing commas after some values in example project file).
I could fix that, but will see if I can make the plugin work on Windows first.

**Comment by rchl** [comment]
And comments that start with '#' which is invalid in preference/project files.

**Comment by davidolrik** [comment]
When you initialise the project you should get a nice config with out syntax errors.
